tests/testthat/test-interact.R

```diff
@@ -102,19 +101,29 @@ test_that("setting path name works", {
    log_remove()
 })
 
-test_that("run_safely_n_quietly works", {
-   scriptPath <- tempfile()
-   logDir <- tempdir()
+test_that("run_safely_loudly works for warnings, errors, messages, and output", {
+   fp <- testthat::test_path("ref", "safely_loudly_test_file.R")
 
-   writeLines("log(-1)
-              log(a)", con = scriptPath)
+   log_config(file = fp)
 
-   log_config(file = scriptPath)
-
-   run_safely_n_quietly(scriptPath)
+   run_safely_loudly(fp)
 
    testthat::expect_true(!is.null(getOption("timber.log")[["warnings"]]))
```

Comment:
Do we need the directs calls to `testthat::` if that package is loaded when you Run the Tests?

Reply:
Not necessarily, we will need to do a cleaning and include checking consistency with stuff like this.  It can be either way we'll just want to pick one at some point.

Reply:
admiral doesn't have these calls so we should remove them to keep in line with what they are doing. I can make a separate issue for us to address?

Reply:
yeah, go ahead and log an issue for that as well as one for checking Roxygen docs for consistency (not sure if Admiral had a style guide or anything)